Lib/logging/__init__.py

```diff
@@ -341,13 +341,10 @@ def __init__(self, name, level, pathname, lineno,
         self.funcName = func
         self.created = ct / 1e9  # ns to float seconds
 
-        # x // 10**n: Drop the rightmost n digits
-        # (x // 10**n) * 10**n: "Change" the rightmost n digits to 0s.
-        # Eg: 55_123_456_789 --> 123_456_789 --> 123_000_000
-        # Lastly divide by 1e6 to change ns to ms
-        # 123_000_000 --> 123
-        # Convert to float by adding 0.0 for historical reasons.
-        self.msecs = (ct - (ct // 1_000_000_000) * 1_000_000_000) // 1_000_000 + 0.0
+        # Get the number of _truncated_ msecs from the nanoseconds.
+        # Eg: 1_677_903_920_999_998_503 ns --> 1_677_903_920_999 ms
```

Comment:
It is not what the code is purposed to calculate, and not how it is implemented. What about the following?
```suggestion
        # Get the number of whole milliseconds (0-999) in the fractional part of seconds.
        # Eg: 1_677_903_920_999_998_503 ns --> 999_998_503 ns -> 999 msec
```

Reply:
Doh! That's what I get for rushing. Sorry for making more work for you! Updated.